preview/MsixCore/msixmgr/Package.cpp

```diff
@@ -148,30 +155,115 @@ HRESULT MsixCoreLib::PackageBase::ProcessPSFIfNecessary()
         TraceLoggingWideString(m_relativeExecutableFilePath.c_str(), "PSF Executable"),
         TraceLoggingWideString(GetPackageDirectoryPath().c_str(), "path"));
 
-    // By default the PSF information is in config.json, but could be in a different json file.
-    for (auto& p : std::experimental::filesystem::directory_iterator(GetPackageDirectoryPath()))
+    // PSF config information should be in a file called config.json, so look for that file
+    std::wstring jsonConfigFile = GetPackageDirectoryPath() + psfConfigFile;
+    bool configFileExists = false;
+    RETURN_IF_FAILED(FileExists(jsonConfigFile, configFileExists));
+
+    if (configFileExists)

```

Comment:
>configFileExists [](start = 8, length = 16)

i'd invert this if (!configFileExists) {TraceLogging that the file is missing, and return not supported},

Reply:
Inverted this, returned,
ERROR_FILE_NOT_FOUND - when config,json is missing,
ERROR_NOT_SUPPORTED - when doc has parse error, runonce = false and endScript is present.
ERROR_NOT_FOUND - when no application id matching manifest is found.